[2022-08-22 02:32:15,767] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.link_dag scheduled__2022-08-18T00:00:00+00:00 [queued]>
[2022-08-22 02:32:15,810] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.link_dag scheduled__2022-08-18T00:00:00+00:00 [queued]>
[2022-08-22 02:32:15,813] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-22 02:32:15,813] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-22 02:32:15,822] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-22 02:32:15,906] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): link_dag> on 2022-08-18 00:00:00+00:00
[2022-08-22 02:32:15,936] {standard_task_runner.py:52} INFO - Started process 17351 to run task
[2022-08-22 02:32:15,953] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'link_dag', 'scheduled__2022-08-18T00:00:00+00:00', '--job-id', '219', '--raw', '--subdir', 'DAGS_FOLDER/testDag.py', '--cfg-path', '/tmp/tmpmd58la9c', '--error-file', '/tmp/tmprhk4df6j']
[2022-08-22 02:32:15,956] {standard_task_runner.py:80} INFO - Job 219: Subtask link_dag
[2022-08-22 02:32:16,255] {task_command.py:371} INFO - Running <TaskInstance: csv_dag.link_dag scheduled__2022-08-18T00:00:00+00:00 [running]> on host 36e1df4dd6ce
[2022-08-22 02:32:16,689] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_dag
AIRFLOW_CTX_TASK_ID=link_dag
AIRFLOW_CTX_EXECUTION_DATE=2022-08-18T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-18T00:00:00+00:00
[2022-08-22 02:33:14,209] {python.py:173} INFO - Done. Returned value was: None
[2022-08-22 02:33:14,226] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=link_dag, execution_date=20220818T000000, start_date=20220822T023215, end_date=20220822T023314
[2022-08-22 02:33:14,270] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-22 02:33:14,315] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
